config/src/main/java/org/polypheny/db/util/PolyMode.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019-2023 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.util;
+
+public enum PolyMode {
```

Comment:
Better call this something like "PolyphenyRunMode" or "RunMode". PolyMode might get confused to be something type or algebra related.

Reply:
RunMode it is.